src/Microsoft.ML.Data/Transforms/TypeConverting.cs

```diff
@@ -539,7 +540,7 @@ public sealed class ColumnInfo
             /// <summary>
             /// The expected kind of the converted column.
             /// </summary>
-            public readonly DataKind OutputKind;
+            internal readonly DataKind OutputKind;
```

Comment:
Why is this internal?

Reply:
It shouldn't be. Thanks.

---
In reply to: [259521046](https://github.com/dotnet/machinelearning/pull/2661#discussion_r259521046) [](ancestors = 259521046)